modules/ROOT/pages/swift.adoc

```diff
@@ -850,6 +850,12 @@ the result.
 Couchbase Mobile 2.0 uses a new replication protocol based on
 WebSockets.
 
+The replicator is designed to send documents from a source to a target database. The target can be one of the following:
+
+- `URLEndpoint`: to replicate data between a local Couchbase Lite database and remote Sync Gateway database using the WebSocket protocol ([start here]).
+- `DatabaseEndpoint`: to replicate data between two local Couchbase Lite databases. This target can be used to create a local back-up of a database ([database replicas])
+- `MessageEndpoint`: allows a local database to replicate with another Couchbase Lite database via a custom transportation protocol such iOS Multipeer Connectivity, Android WiFi Direct, Android NearByConnection, socket based transportation etc. (see xref:#message-endpoint[Message Endpoint])
+
```

Comment:
Call out EE only for the above two?

Reply:
Indeed, I've pushed a commit to point this out for encryption and the message endpoint API. ([preview](http://docs-build.sc.couchbase.com/antora/site/couchbase-lite/2.1/swift.html#message-endpoint))